packages/@okta/vuepress-site/docs/reference/releases-at-okta/index.md

```diff
@@ -1,71 +1,72 @@
 
-Documentation for features in Beta release is marked with the Beta icon: <ApiLifecycle access="beta" />
+* **Beta (Self-Service):** Self-directed without ongoing support or regular contact with Okta. Contact is primarily limited to collecting feedback. If you are interested in a Beta Self-Service feature, you can enable that feature for your org from the **Settings** > **Features** page in the Admin Console.
 
-## Early Access (EA)
+You can track the availability of Beta features on the [Product Roadmap](https://support.okta.com/help/s/productroadmap) page.
 
-A feature in an Early Access (EA) release is new or enhanced functionality made available for customers to selectively "opt-in" to and use in both Production and non-Production environments.
+See [Manage Early Access and Beta features](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Manage_Early_Access_features) for more information on Okta's Self-Service Feature Manager.
 
-Features in EA:
+## Early Access (EA)
 
-* Are supported by Okta Customer Support.
-* Obey the SLAs. APIs are [backwards compatible](/docs/reference/api-overview/).
+A feature in an Early Access (EA) stage is new or enhanced functionality made available for you to selectively opt-in to and use in both Production and non-Production environments.
 
->Note: A feature may skip EA if it doesn't affect other features or functionality.
-Also, although we strive for API-first development, a feature in EA may add functionality later.
+> **Note:** A feature may skip EA if it doesn't affect other features or functionality. Also, although we strive for API-first development, a feature in EA may add functionality later.
 
 Any bug fixes or improvements are managed and fixed with the same timeline and processes as those in General Availability.
 
-Features in EA release are marked with the EA icon: <ApiLifecycle access="ea" />
+Features in EA release are marked with the EA icon: <Apilife cycle access="ea" />
+
+EA releases are managed in two ways:
+
+* **Early Access:** If you are interested in an EA feature, you must contact Support to enable the feature. Support can enable features in EA for any customer, provided any required pre-requisites have been met.
+
+* **Early Access (Self-Service):** If you are interested in an EA Self-Service feature, you can enable the feature for your org from the **Settings** > **Features** page in the Admin Console.
+
+You can track the availability of EA features on the [Product Roadmap](https://support.okta.com/help/s/productroadmap) page.
+
+See [Manage Early Access and Beta features](https://help.okta.com/en/prod/okta_help_CSH.htm#ext_Manage_Early_Access_features) for more information on Okta's Self-Service Feature Manager.
 
 ## General Availability (GA)
```

Comment:
For GA, I know this is not part of this pull request, but the existing content might be unclear. You could read it as saying that all EA features consistently spend just one month in EA and then move to GA.

Reply:
that's a good point. I'll ask Beth about that. Thanks.

Reply:
@bethwang-okta wants to leave the wording as is.